mycroft/skills/intent_service.py

```diff
@@ -13,22 +13,21 @@
 # limitations under the License.
 #
 """Mycroft's intent service, providing intent parsing since forever!"""
-from copy import copy
 import time
+from threading import Event
 
 from mycroft.configuration import Configuration, setup_locale
-from mycroft.util.log import LOG
-from mycroft.util.parse import normalize
+from mycroft.messagebus.message import Message
```

Comment:
Should this import from `mycroft_bus_client`?

Reply:
no, in core Message is patched to have utterance_remainder method